products/memgraph/v0.51.0/reference_guide/query-modules.md

```diff
@@ -6,35 +6,30 @@ startup.
 
 ### Loading Query Modules
 
-If you wish to use procedures from query modules, you have to tell Memgraph
-where to find them. This is done when running Memgraph using the
-`--query-modules-directory` command line flag.
-
 The Memgraph installation comes with the `example.so` and `py_example.py` query
-module. We will use them to explain how query modules work. You should run
-Memgraph with `--query-modules-directory` pointing to Memgraph's installation
-`lib/memgraph/query_modules` directory. Assuming the standard installation on
-Debian, you would run Memgraph with the following command.
+modules which are located in `usr/lib/memgraph/query_modules` directory,
+Assuming the standard installation on Debian, you would run Memgraph with the
+following command.
 
 ```plaintext
-/usr/lib/memgraph/memgraph --query-modules-directory=/usr/lib/memgraph/query_modules
+systemctl start memgraph
 ```
 
 When using Docker, the equivalent would be the following.
 
 ```plaintext
 docker run -p 7687:7687 \
   -v mg_lib:/var/lib/memgraph -v mg_log:/var/log/memgraph -v mg_etc:/etc/memgraph \
-  memgraph --query-modules-directory=/usr/lib/memgraph/query_modules
+  memgraph
 ```
 
 Memgraph will now attempt to load the query modules form all `*.so` and `*.py`
-files it finds in the given directory. The `*.so` modules are written
-using the C API and the `*.py` modules are written using the Python API. Each
-file corresponds to one query module. Names of these files will be mapped to
-query module names.  So in our case, we have an `example.so` which will be
-mapped to `example` module and a `py_example.py` which will be mapped to
-`py_example` module in the query language.
+files it finds in the default (`usr/lib/memgraph/query_modules`) directory.
+The `*.so` modules are written using the C API and the `*.py` modules are
+written using the Python API. Each file corresponds to one query module. Names
+of these files will be mapped to query module names.  So in our case, we have
+an `example.so` which will be mapped to `example` module and a `py_example.py`
+which will be mapped to `py_example` module in the query language.
 
 Each query module can define multiple procedures. Both of our examples define
 a single procedure creatively named `procedure`.
```

Comment:
I would add the following paragraph below this one:

If you want to change the directory Memgraph searches for query modules, just change the `--query-modules-directory` flag in the main configuration file (`/etc/memgraph/memgraph.conf`) or supply it as a command-line parameter (e.g. when using Docker).